src/main/scripts/org/reaktivity/specification/nukleus/amqp/streams/send.to.client.when.fragmented/client.rpt

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright 2016-2020 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property message600 ${amqp:randomBytes(600)}
+
+connect await ROUTED_CLIENT
+        "nukleus://streams/amqp#0"
+        option nukleus:window 8192
+        option nukleus:transmission "duplex"
+
+write nukleus:begin.ext ${amqp:beginEx()
+                              .typeId(nukleus:id("amqp"))
+                              .address("clients")
+                              .capabilities("RECEIVE_ONLY")
+                              .senderSettleMode("SETTLED")
+                              .receiverSettleMode("FIRST")
+                              .build()}
+
+read nukleus:begin.ext ${amqp:beginEx()
+                             .typeId(nukleus:id("amqp"))
+                             .address("clients")
+                             .capabilities("RECEIVE_ONLY")
+                             .senderSettleMode("SETTLED")
+                             .receiverSettleMode("FIRST")
+                             .build()}
+connected
+
+read nukleus:data.ext ${amqp:dataEx()
+                            .typeId(nukleus:id("amqp"))
+                            .deferred(600 - 155)
+                            .deliveryId(0)
+                            .deliveryTag("0")
+                            .messageFormat(0)
+                            .flags("SETTLED")
+                            .build()}
+
+read ${message600}
```

Comment:
Avoid sharing properties across both sides of the same scenario, as it forces redefinition of the same property in both scripts in case only one script is used.

Instead, be more lenient on the read side, by reading a fixed number of bytes (600 in this case), also without capturing into a variable.

This will allow the `message600` property to be removed from this script.

This feedback applies to other similar scenario scripts too.